flink-shaded-hadoop/pom.xml

```diff
@@ -111,6 +111,7 @@ under the License.
 									<include>io.netty:netty:*</include>
 									<include>org.apache.curator:*</include>
 									<include>org.apache.hadoop:*</include>
+									<include>org.mortbay.jetty:*</include>
```

Comment:
It is a point solution only for webhdfs API. In code path of webhdfs API, hadoop classes call jetty-util classes. Since jetty-utils classes are not available in "flink class loader", call fails. Even if I pack jetty-util classes in my application jar, call still fails since application jar is in "different classloader". There are two ways to solve -
1. Relocate hadoop classes in flink fat jar. Now the webhdfs call will go through hadoop classes packed in application jar, and not the flink jar.
2. Include jetty-util classes in flink fat jar. This is what I am doing in this PR. 

Hope I have clarified the problem